maintainers/maintainer-list.nix

```diff
@@ -18932,6 +18932,12 @@
       fingerprint = "FD0A C425 9EF5 4084 F99F 9B47 2ACC 9749 7C68 FAD4";
     }];
   };
+  yelite = {
+    name = "Lite Ye";
+    email = "yelite958@gmail.com";
+    github = "yelite";
+    githubId = 3517225;
+  };
```

Comment:
Could you split this of into a separate commit titled `maintainers: add yelite`, preceeding the `init at` commit